### PR TITLE
Update to ARFoundation documentation

### PR DESCRIPTION
**Pull request by CDiaz-MS**
## Overview
Added a guide on how to add support for Android and iOS with screenshots and updated the table of contents.

Addresses issue #6646 at the end of UsingARFoundation.md

## Changes
- Added CrossPlatform folder to docs, updated in toc.yml
- Added UsingARFoundation.md
- Updated UnityARCameraSettings.md

### CODE REVIEW

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+2. In the Unity Package Manager (UPM), install the following packages:
+
+    ### Unity 2018.4.x 
+
+    | **Android** | **iOS** | Comments |
+    | --- | --- | --- |
+    | AR Foundation  <br/> Version: 1.5.0 - preview 6 | AR Foundation  <br/> Version: 1.5.0 - preview 6 | For Unity 2018.4, this package is included as a preview. To view package: Window > Package Manager > Advanced > Show Preview Packages|
+    | ARCore XR Plugin <br/> Version: 2.1.2 | | |
```
Comment (julenka) [inline Documentation/CrossPlatform/UsingARFoundation.md]: This table formatted incorrectly for me in the preview

File: Documentation/CrossPlatform/UsingARFoundation.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    |  | ARKit XR Plugin <br/> Version: 2.1.2  | |
+
+    ### Unity 2019.x 
+
+    | **Android** | **iOS** | 
+    | --- | --- |
+    | AR Foundation  <br/> Version: 2.1.4 |  AR Foundation  <br/> Version: 2.1.4 |
+    | ARCore XR Plugin <br/> Version: 2.1.2 | |
```
Comment (julenka) [inline Documentation/CrossPlatform/UsingARFoundation.md]: This table also formatted incorrectly in preview

**Comment by CDiaz-MS** [comment]
>  Are there any workarounds needed for iOS currently?

@julenka 
The only current workaround is mentioned at the bottom of the UsingARFoundation page to prevent the World Anchor error in Xcode

File: Documentation/CrossPlatform/UsingARFoundation.md (hunk 1 above)
Comment (CDiaz-MS) [reply Documentation/CrossPlatform/UsingARFoundation.md]: The blank cells in the table were intentional to indicate that ARKit is not needed for Android and ARCore is not needed for iOS.  But if it looks like formatting issue then I can change the way the packages and versions are listed for each platform and unity version.

Comment (julenka) [reply Documentation/CrossPlatform/UsingARFoundation.md]: This is what I see, 

image

Ah, after staring at it for about 30 seconds more, I'm still not sure what the table is telling me :-) Would recommend maybe changing the visualization. Maybe just a bulletted list, one for android, one for iOS?

**Pull request merged**